### PR TITLE
Removed some bugs while yaml creation

### PR DESCRIPTION
**Pull request by swati-kale**
Signed-off-by: Swati Kale <swkale@redhat.com>

### CODE REVIEW

File: frontend/src/component/operator-wizard/page-component/Page.js [hunk 1]
```diff
@@ -128,8 +133,19 @@ export default class Page extends Component {
...
-    //alert("deploy here");
-    this.createSampleYaml();
+    var sampleYaml = this.createSampleYaml();
+    var result = this.createResultYaml(sampleYaml);
+    console.log(result);
+    fetch("/", {
+      method: "POST",
+      body: JSON.stringify(result),
+      headers: {
+        "Content-Type": "application/json"
```
Comment (bmozaffa) [inline frontend/src/component/operator-wizard/page-component/Page.js]: Are we posting json or yaml?

File: frontend/src/component/operator-wizard/page-component/Page.js [hunk 2]
```diff
@@ -248,6 +288,15 @@ export default class Page extends Component {
     return (
       <Form id={"form-page-" + this.props.pageNumber}>
         <div key={"page" + this.props.pageNumber}>
+          <Button
+            key="try"
+            variant="secondary"
```
Comment (bmozaffa) [inline frontend/src/component/operator-wizard/page-component/Page.js]: Is this a test? should it be merged?

File: frontend/src/component/operator-wizard/page-component/Page.js (hunk 1 above)
Comment (swati-kale) [reply frontend/src/component/operator-wizard/page-component/Page.js]: i didnot look into deploy part.
As per Karel's comment, just moved this code from editYaml to Deploy functionality.

Comment (swati-kale) [reply frontend/src/component/operator-wizard/page-component/Page.js]: result -consists of Yaml  thouh.

Comment (bmozaffa) [reply frontend/src/component/operator-wizard/page-component/Page.js]: ok, so look at how this file may have changed as a result of my previous PR, and reapply your changes accordingly.

File: frontend/src/component/operator-wizard/page-component/Page.js (hunk 2 above)
Comment (swati-kale) [reply frontend/src/component/operator-wizard/page-component/Page.js]: i will take the latest , please donot merge.

**Pull request closed without merging**